foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/util/YamlUtils.java

```diff
@@ -78,23 +97,23 @@ public String getString(String... path) {
         }
 
         public boolean hasInt(String... path) {
-            return hasValue(int.class, path);
+            return hasValue(Integer.class, path);
```

Comment:
I: these were returning false when they shouldn't have as the map has the boxed version of the value so `isInstance` was failing.